managed/src/main/java/com/yugabyte/yw/models/helpers/CustomerConfigValidator.java

```diff
@@ -179,4 +191,44 @@ public ObjectNode validateDataContent(JsonNode formData) {
         formData.get("data"), errorJson));
     return errorJson;
   }
+
+  /**
+   * Validates data which is contained in formData.
+   * During the procedure it checks all the S3 config parameters whether they are valid or not.
+   * Errors are collected and
+   * returned back as a result. Empty result object means no errors.
+   *
+   * Currently is checked:
+   *  - S3/AWS - S3 Bucket, whether it exists or not.
+   *
+   * @param formData, customerUUID
+   * @return Json filled with errors
+   */
+  public ObjectNode validateS3DataContent(JsonNode formData, UUID customerUUID) {
+    ObjectNode errorJson = Json.newObject();
+    try {
+      AWSCredentials credentials = new BasicAWSCredentials(
+        formData.get("data").get("AWS_ACCESS_KEY_ID").asText(),
+        formData.get("data").get("AWS_SECRET_ACCESS_KEY").asText()
+      );
+      String region = Region.getByProvider(Provider.get(customerUUID, CloudType.aws).uuid)
+        .get(0).code.replace('-', '_').toUpperCase();
```

Comment:
Check how I convert to aws region in getEC2Client.
I do not recall having to do any string manipulation.  But may be I misremember

Reply:
Here I had to do this because from the backend we are getting region something like this **ap-south-1** which ideally should be like "ap_south_1".

In your case, we might be getting region details in the correct format i.e **ap_south_1**